.circleci/config.yml

```diff
@@ -40,6 +40,7 @@ commands:
       - run:
           name: Install libraries for GPU tests
           command: |
+            sudo apt update
```

Comment:
`apt update` is not the same as `apt-get update`

Reply:
Oh wow, I didn't knew that. Though could it still be that the `apt update` is enough? I personally trying to keep the CI clean while I'm at it as usually I tend to copy and paste things to other jobs, and as more creep there is the worse it gets over time.

Reply:
That I don't know 100%.  In my experience, `apt-get update` is for normal system updates, but since debian/ubuntu are more or less rolling update systems, there are times when the sources `apt-get` uses are updated.  That's where `apt update` comes in handy.  I personally always use both when needed (although admittedly it's rare that I use `apt update`).  Since we expect an automated system to always be updating, it's safe to run both for those times when needed.

Since we're updating, I think it's a good idea to always just do both so this doesn't bite us again in the future.

Reply:
I had to resolve the same issue in the FFI repo btw, although that was a little while ago.  It's been working nicely there.